docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,11 +1,21 @@
-[id='pro-running-the-published-repositories-{chapter}']
-= Introduction
+[id='{context}-pro-running-the-demo-repositories']
+= Running the Demo App from the Published Repositories
```

Comment:
`Running the {RC-NAME_Short} applications on local machine `

Reply:
I'm going to add a variable here so you can call them what you want when you/Eng have made a decision on what to call them.

FYI - The original heading from Eng was "Running the Published Repositories for the Demo App".